### PR TITLE
JSLIGO: update syntax, content

### PR DESCRIPTION
**Pull request by lykimq**

### CODE REVIEW

File: src/frontend/src/pages/Chapters/JS/ChapterVariables/course.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-<dialog character="mechanics">Seems like you have locked-in the parameters. You should unlock them so we can later change the ship configuration if need be.</dialog>
+<dialog character="mechanics">Seems like you have locked in the parameters. You should unlock them so we can later change the ship configuration if need be.</dialog>
 
 ## Constants
 
-Constants are immutable by design, which means their values cannot be reassigned. Put in another way, they can be assigned once, at their declaration. When defining a constant you need to provide a name, type and a value:
+Constants are immutable by design, which means their values cannot be reassigned. Put another way, they can be assigned once, at their declaration. When defining a constant, you need to provide a name, type, and a value:
 
 ```
 const age: int = 25;
```
Comment (Laucans) [inline src/frontend/src/pages/Chapters/JS/ChapterVariables/course.md:10]: In many cases ";" is not mandatory. We don't want that people think it is so I would like to remove them

File: src/frontend/src/pages/Chapters/JS/ChapterVariables/course.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ Variables, unlike constants, are mutable. They cannot be declared in a global sc
 let add = (a: int, b: int): int => {
```
Comment (Laucans) [inline src/frontend/src/pages/Chapters/JS/ChapterVariables/course.md]:
```suggestion
const add = (a: int, b: int): int => {
```

File: src/frontend/src/pages/Chapters/JS/ChapterMath/course.md [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
 ```
 
-⚠️ You cannot add a tez and a int.
+⚠️ You cannot add a tez and an int.
 
 ℹ️ You can use underscores for readability when defining large numbers:
 
 ```
-let sum : tez = 100_000 as mutez;
+let sum : tez = 100_000mutez;
```
Comment (Laucans) [inline src/frontend/src/pages/Chapters/JS/ChapterMath/course.md]:
```suggestion
const sum : tez = 100_000mutez;
```

File: src/frontend/src/pages/Chapters/JS/ChapterVariables/solution.jsligo [hunk 4]
```diff
@@ -1,4 +1,4 @@
 // Type your solution below
 type ship_code = string;
-let my_ship: ship_code = "020433";
+const my_ship: ship_code = "020433";
```
Comment (Laucans) [inline src/frontend/src/pages/Chapters/JS/ChapterVariables/solution.jsligo]:
```suggestion
let my_ship: ship_code = "020433";
```
There is an update after. So it's a variable not a constant

Comment (Laucans) [reply src/frontend/src/pages/Chapters/JS/ChapterVariables/solution.jsligo]: Or maybe not, at the end IDK why there is an update on top level, which is not possible in Ligo
image

**Review by Laucans — COMMENTED** [review]
Can you take my first comments in consideration and test every solutions onto a jsligo file to be sure it compile  + no warnings (except if it's necessary du to the tutorial of course)? 
Looks like it's not the case of every snippets. 
Also you can check if snippets compiles with REPL. Or integrate [ligo-mdx ](https://github.com/ligolang/ligo-mdx)
image

File: src/frontend/src/pages/Chapters/JS/ChapterVariables/course.md (hunk 1 above)
Comment (lykimq) [reply src/frontend/src/pages/Chapters/JS/ChapterVariables/course.md:10]: I remove it at this [commit](https://github.com/ligolang/tezos-academy/pull/5/commits/7bd434d5e5393eb363057ec2a9e713265a87d2e7)

File: src/frontend/src/pages/Chapters/JS/ChapterVariables/course.md (hunk 2 above)
Comment (lykimq) [reply src/frontend/src/pages/Chapters/JS/ChapterVariables/course.md]: I fix it at this [commit](https://github.com/ligolang/tezos-academy/pull/5/commits/7bd434d5e5393eb363057ec2a9e713265a87d2e7)

File: src/frontend/src/pages/Chapters/JS/ChapterVariables/solution.jsligo (hunk 4 above)
Comment (lykimq) [reply src/frontend/src/pages/Chapters/JS/ChapterVariables/solution.jsligo]: For the `let` I return it at this [commit](https://github.com/ligolang/tezos-academy/pull/5/commits/7bd434d5e5393eb363057ec2a9e713265a87d2e7)

File: src/frontend/src/pages/Chapters/JS/ChapterMath/course.md (hunk 3 above)
Comment (lykimq) [reply src/frontend/src/pages/Chapters/JS/ChapterMath/course.md]: I fix it at this [commit](https://github.com/ligolang/tezos-academy/pull/5/commits/5a5773381fa92e37928f0f7e4772c8b0898fbc8c)

**Comment by Laucans** [comment]
Mdx integration is badly done. Please check the Readme to integrate it correctly

File: src/frontend/src/pages/Chapters/JS/ChapterVariables/course.md (hunk 1 above)
Comment (Laucans) [reply src/frontend/src/pages/Chapters/JS/ChapterVariables/course.md:10]: Talking about ; on every files. not only this one

**Pull request merged**